common/data_refinery_common/models/models.py

```diff
@@ -108,13 +124,21 @@ def __str__ (self):
     publication_doi = models.CharField(max_length=64, blank=True)
     pubmed_id = models.CharField(max_length=32, blank=True)
     source_first_published = models.CharField(max_length=256, blank=True)
-    source_last_updated = models.CharField(max_length=256, blank=True)
+    source_last_modified = models.CharField(max_length=256, blank=True)
```

Comment:
Should we try to parse these into actual times?

With charfields we can't do queries like, give me all samples original made after 2013, etc..

Reply:
I want to say yes. My fear is that at some point we're gonna get some bad data from the external sources which isn't nicely parsable... Do we trust them? Is a badly-formatted data a good reason to skip an experiment? Once we make this a date or datetime field we'll have less options once we do run into a bad string.

I'm pretty balanced on the fence about this, so nudge me either way and I'll make it happen.